distros/kubernetes/nvsentinel/charts/gpu-health-monitor/templates/configmap.yaml

```
@@ -36,3 +36,4 @@ data:
     EnabledEventProcessors = PlatformConnectorEventProcessor
 
 {{ (.Files.Glob "files/dcgmerrorsmapping.csv").AsConfig | indent 2 }}
+
```

YAMLlint (1.38.0)

[warning] too many blank lines (1 > 0)

(empty-lines)